cmd/operator-sdk/exec-entrypoint/cmd.go

```diff
@@ -12,20 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package up
+package run
```

Comment:
```suggestion
package execentrypoint
```

Ans should we rename the directory to match the package name for consistency?